src/arduino/arduinoSettings.ts

```diff
@@ -53,7 +53,17 @@ export class ArduinoSettings implements IArduinoSettings {
             } else {
                 this._packagePath = path.join(process.env.HOME, ".arduino15");
             }
-            this._sketchbookPath = this.preferences.get("sketchbook.path") || path.join(process.env.HOME, "Arduino");

```

Comment:
combine them to a function ?

Reply:
The second part, here is `path.join(process.env.HOME, "Arduino")`, is related with platform, so I do not think we can get any benefit from combining them to a function